samples/search/autosuggest_samples.py

```diff
@@ -9,19 +9,23 @@
 
 SUBSCRIPTION_KEY_ENV_NAME = "AUTOSUGGEST_SUBSCRIPTION_KEY"
 
+
 def autosuggest_lookup(subscription_key):
     """AutoSuggestLookup.
 
     This will look up a single query (Xbox) and print out name and url for first web result.
     """
-    client = AutoSuggestSearchAPI(CognitiveServicesCredentials(subscription_key))
+    client = AutoSuggestSearchAPI(
+        CognitiveServicesCredentials(subscription_key))
 
     try:
-        suggestions = client.auto_suggest(query="Satya Nadella")  # type: Suggestions
+        suggestions = client.auto_suggest(
+            query="Satya Nadella")  # type: Suggestions
 
         if suggestions.suggestion_groups:
             print("Searched for \"Satya Nadella\" and found suggestions:")
-            suggestion_group = suggestions.suggestion_groups[0]  # type: SuggestionsSuggestionGroup
+            # type: SuggestionsSuggestionGroup
+            suggestion_group = suggestions.suggestion_groups[0]
```

Comment:
type annotations are required to be after the line, might be a autopep8 issue

Reply:
Seems like it. Also we're going to be trying to move all samples to be compatible with Python 3.6 so official type hinting support should be used.

Reply:
@sptramer if you do so, that would mean 2.7 or even 3.5 people won't be able to execute the sample, is that really what we want???

Reply:
Yes. Python 2 is being sunsetted permanently on Jan 1 2020, and research indicates that the vast majority of Python users are on 3.6.

Reply:
Research maybe but not metrics :). Please discuss this point with my PM, I'll start an email thread

Reply:
So, I don't want to let your PR open too long, I don't you to have conflict to solve. I started an email thread to discuss 2.7 support, in the meantime I revert this particular so I can merged it. If you feel really strong about this particular, line, feel free to open a new PR with just this line.
Thanks